Set4/29/parser/parser.h

```diff
@@ -0,0 +1,32 @@
+#ifndef INCLUDED_PARSER_
+#define INCLUDED_PARSER_
+#include "../line.h"
+#include <iosfwd>
+#include <string>
+
+class Parser
+{
+    Line d_line;
+    bool d_integral;
+    
+    public:
+        Parser();
+        
+        enum Return                     // To keep track of data type
+        {
+            NO_NUMBER,
+            NUMBER,
+            EOLN
+        };
+        
+        bool reset();                   // Fills d_line with next input 
+        Return number(double *dest);    // Puts d_line's next phrase in dest
+        bool isIntegral();              // Checks if value is integral/integer
```

Comment:
```suggestion
        bool isIntegral() const;              // Checks if value is integral/integer
```

(also in implementation)